Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCardModel.swift

```diff
@@ -269,9 +281,9 @@ extension PPOProjectCardModel {
   internal static let completeSurveyTemplate = PPOProjectCardModel(
     isUnread: true,
     alerts: [
-      .init(type: .alert, icon: .warning, message: "Survey available")
+      .init(type: .warning, icon: .alert, message: "Survey available")
     ],
-    imageURL: URL(string: "http://localhost/")!,
+    imageURL: URL(string: "https://placecats.com/400/200")!,
```

Comment:
Are we using hardcoded url intentionally here? If placecats.com gets down, will it be problem ?

Reply:
Removed

Reply:
Thanks